installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh

```diff
@@ -119,14 +50,31 @@ sed -i "s/{inst_stack_prefix}/$stackprefix/g" ./jazz-core/serverless-config-pack
 sed -i "s/{inst_stack_prefix}/$stackprefix/g" ./jazz-core/serverless-config-pack/serverless-nodejs.yml
 sed -i "s/{inst_stack_prefix}/$stackprefix/g" ./jazz-core/serverless-config-pack/serverless-python.yml
 
-#Adding platform_usermanagement value injection
-sed -i "s/^securityGroupIds.*.$/securityGroupIds: $securityGroupIds/g" ./jazz-core/platform_usermanagement/deployment-env.yml
-sed -i "s/^subnetIds.*.$/subnetIds: $subnetIds/g" ./jazz-core/platform_usermanagement/deployment-env.yml
-sed -i "s=^iamRoleARN.*.$=iamRoleARN: $iamRoleARN=g" ./jazz-core/platform_usermanagement/deployment-env.yml
-sed -i "s/^region.*.$/region: $region/g" ./jazz-core/platform_usermanagement/deployment-env.yml
-
-#Adding platform_email value injection
-sed -i "s/^securityGroupIds.*.$/securityGroupIds: $securityGroupIds/g" ./jazz-core/platform_email/deployment-env.yml
-sed -i "s/^subnetIds.*.$/subnetIds: $subnetIds/g" ./jazz-core/platform_email/deployment-env.yml
-sed -i "s=^iamRoleARN.*.$=iamRoleARN: $iamRoleARN=g" ./jazz-core/platform_email/deployment-env.yml
-sed -i "s/^region.*.$/region: $region/g" ./jazz-core/platform_email/deployment-env.yml
+#-------------------------------------------
+platform_services=("cognito-authorizer" "logs" "usermanagement" "services-handler" "events" "services" "logout" "login" "cloud-logs-streamer" "is-service-available" "delete-serverless-service" "create-serverless-service" "email" )
+servicename="_services_prod"
+tablename=$stackprefix$servicename
+for element in "${platform_services[@]}"
+do
+  uuid=`uuidgen -t`
+  aws dynamodb put-item --table-name $tablename --item '{
+          "SERVICE_ID":{"S":"'$uuid'"},
+          "SERVICE_CREATED_BY":{"S":"'$jazz_admin'"},
+          "SERVICE_DOMAIN":{"S":"platform"},
+          "SERVICE_NAME":{"S":"'$element'"},
+          "SERVICE_RUNTIME":{"S":"nodejs"},
+          "SERVICE_STATUS":{"S":"active"},
+          "SERVICE_METADATA":{"M":{
+                          "securityGroupIds":{"S":"'$securityGroupIds'"},
+                          "subnetIds":{"S":"'$subnetIds'"},
+                          "iamRoleARN":{"S":"'$iamRoleARN'"},
+                          "providerMemorySize":{"S":"256"},
+                          "providerRuntime":{"S":"nodejs4.3"},
+                          "providerTimeout":{"S":"160"},
+                          "runtime":{"S":"nodejs"},
+                          "service":{"S":"'$element'"},
```

Comment:
service is not part of meta-data, its already part of the name.

Reply:
Removed the service from metadata